using-packages.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: page
+title: Using Packages
+permalink: /using-packages/
+---
+
+Flutter supports using shared packages contributed by other developers to the
+Flutter and Dart ecosystems. This allows you to quickly build your app without
+having to develop everything from scratch.
+
+Existing packages enable many use cases, for example, file path manipulation
+([`path`](https://pub.dartlang.org/packages/path)), custom navigation/route
+handling ([`fluro`](https://pub.dartlang.org/packages/fluro)), integration
+with device APIs (like
+[`url_launcher`](https://pub.dartlang.org/packages/url_launcher) &
+[`battery`](https://pub.dartlang.org/packages/battery)), and using third-party platform
+SDKs (like
+[Firebase](https://github.com/flutter/plugins/blob/master/FlutterFire.md)).
+
+* TOC
+{:toc}
+
+## Using packages
+
+### Searching for packages
+
+Packages are published to the *[Pub](https://pub.dartlang.org)* package repository. 
+
+The list [Flutter packages](https://pub.dartlang.org/flutter/packages) displays
+packages that have been developed specifically for Flutter (i.e., that declare a
+dependency on the Flutter framework). Flutter-specific packages are presented
+with a Flutter logo adornment.
+
+The standard Pub search field supports searching among all published packages.
+Note however that many packages depend on other frameworks, such as the
+[AngularDart](https://webdev.dartlang.org/angular/) web framework, and those
+will not work with Flutter. We hope to offer a better search experience with
+support for filtering to only packages that are compatible with Flutter in the
+future.
+
+{% include note.html content="Flutter is still a young platform, and only a
+fairly small set of Flutter packages are currently available on pub. We
+encourage all Flutter developers to [develop](/developing-packages/) new
+packages." %}
+
+### Adding a package dependency to an app
+
+To add a plugin 'plugin1' to an app:
+
+1. Open the `pubspec.yaml` file located inside your app folder, and add
+`plugin1:` under `dependencies`.
+
+1. Get the plugin:
+   * From the terminal: Run `flutter packages get`
+   * From IntelliJ: Click 'Packages Get' in the action ribbon at the top of `pubspec.yaml`
+
+1. Build or run your app. As part of this, Flutter will add the package code to
```

Comment:
This is a little confusing. The code was already added to my add when I installed ("get") the package.

I'm not sure we need this last bullet. Only two steps! :)

maybe, Step Three is "import the package's library into your app"